File: 00_ETC_PY/17350.py
n = int(input())
is_Anj = False
for i in range(n):
    s = input()
    if s == "anj":
        is_Anj = True

if is_Anj == True:
    print("뭐야;")
else:
    print("뭐야?")

File: 00_ETC/17350.cpp
#include <iostream>
using namespace std;

int main() {
    cin.tie(0)->sync_with_stdio(0);

    int n;
    cin >> n;
    bool isAnj = false;
    for (int i = 0; i < n; ++i) {
        string s;
        cin >> s;
        if (s == "anj") {
            isAnj = true;
        }
    }
    if (isAnj == true) {
        cout << "뭐야;";
    } else {
        cout << "뭐야?";
    }
    return 0;
}
//
// Created by 최승우 on 25. 9. 13.
//
